doc/modules/preprocessing.rst

```diff
@@ -146,6 +148,62 @@ full formula is::
 
     X_scaled = X_std / (max - min) + min
 
+:class:`MaxAbsScaler` works in a very similar fashion, but scales data so
+it lies within the range ``[-1, 1]``, and is meant for data
+that is already centered at zero. In particular, this scaler is very well
+suited for sparse data.
```

Comment:
Well sparse data is not necessarily centered at zero. I would say data that is already centered or sparse data. And then say that centering sparse data would destroy sparsity.